activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Add `bigdecimal` as Active Support dependency that is a bundled gem candidate for Ruby 3.4.
```

Comment:
Should we move this into the `.beta.1` changelog since that was when it was actually added?

Reply:
Right. Moved this entry into the `Rails 7.1.0.beta1`.